.github/workflows/test-deploy-macos.yml

```diff
@@ -0,0 +1,23 @@
+name: Test set-up on MacOS
+
+on:
+  push:
+    branches:
+      - 'master'
+  pull_request:
+    branches:
+      - 'master'
+
+jobs:
+  build:
+
+    runs-on: macos-latest
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set-up Docker
+      run: |
+        make install-docker-osx
```

Comment:
Seems `brew cask install docker` does not install `docker-compose` any more or it does not make it available in the path right away. We might want to add `brew install docker-compose` on line 43 of the `Makefile`.

Reply:
Even though I added `brew install docker-compose` and this worked, I'm still getting the following error on the MacOS checks:
```logfile
cp .env.dist .env
make build
docker-compose build --pull
Couldn't connect to Docker daemon. You might need to install Docker:

https://docs.docker.com/engine/installation/mac/
make[1]: *** [build] Error 1
make: *** [init-env] Error 2
##[error]Process completed with exit code 2.
```